client/src/templates/Introduction/components/CertChallenge.js

```diff
@@ -82,6 +102,10 @@ const CertChallenge = ({
   const { certSlug } = certMap.find(x => x.title === title);
 
   useEffect(() => {
+    if (complete && !pending) {
+      setUserLoaded(true);
+    }
```

Comment:
Take this with a grain of salt:

As `complete` and `pending` are props, we should be adding them to the dependency array of **their own** `useEffect`. Also, I do not think they should be _accessed_ (destructured) within the root of the function - should be destructured within the `useEffect` callback.

Reply:
Thanks for the review 👍